_docs/install/health.md

```diff
@@ -43,18 +44,15 @@ $ curl -v http://localhost:5705/v1/health
 < Content-Length: 539
 <
 * Connection 0 to host localhost left intact
-{"submodules":{"kv":{"status":"alive","message":"","updatedAt":"2017-08-11T12:07:55.503707066Z","changedAt":"2017-08-11T11:31:41.124110346Z"},"kv_write":{"status":"alive","message":"","updatedAt":"2017-08-11T12:07:55.503707825Z","changedAt":"2017-08-11T11:31:51.088848317Z"},"nats":{"status":"alive","message":","updatedAt":"2017-08-11T12:07:55.503706574Z","changedAt":"2017-08-11T11:31:51.088844662Z"},"scheduler":{"status":"alive","message":"","updatedAt":"2017-08-11T12:07:55.503707417Z","changedAt":"2017-08-11T11:31:51.088847582Z"}}}
+{"submodules":{"kv":{"status":"alive","message":"","updatedAt":"2017-08-16T14:24:59.898288145Z","changedAt":"2017-08-16T13:06:18.672362683Z"},"kv_write":{"status":"alive","message":"","updatedAt":"2017-08-16T14:24:59.898289093Z","changedAt":"2017-08-16T13:06:27.475859537Z"},"nats":{"status":"alive","message":"","updatedAt":"2017-08-16T14:24:59.898287588Z","changedAt":"2017-08-16T13:06:27.475858077Z"},"scheduler":{"status":"alive","message":"","updatedAt":"2017-08-16T14:24:59.898288556Z","changedAt":"2017-08-16T13:06:27.475859095Z"}}}
 ```
 
 ### Install logs
 
 For container installs (`storageos/node`), pipe the container logs to grep to check that the cluster services initialized correctly:
 
 ```bash
-$ docker ps
-CONTAINER ID        IMAGE                          COMMAND              CREATED             STATUS                    PORTS               NAMES
-f01c6551b5eb        storageos/node:latest   "storageos server"   35 minutes ago      Up 35 minutes (healthy)                       storageos
-$ docker logs f01c6551b5eb  2>/dev/null | grep StorageOS
+$ docker logs storageos 2>/dev/null | grep StorageOS
```

Comment:
Think you should remove the `2>/dev/null | grep StorageOS` here.  `docker logs storageos` will give only the StorageOS logs.